Library/brew.sh

```diff
@@ -209,10 +209,13 @@ EOS
   esac
 fi
 
-# Hide shellcheck complaint:
-# shellcheck source=/dev/null
-source "$HOMEBREW_LIBRARY/Homebrew/utils/ruby.sh"
-setup-ruby-path
+if [[ -z "$HOMEBREW_BASH_COMMAND" ]]
+then
+  # Hide shellcheck complaint:
+  # shellcheck source=/dev/null
+  source "$HOMEBREW_LIBRARY/Homebrew/utils/ruby.sh"
+  setup-ruby-path
+fi
```

Comment:
Or we can avoid using `ENV/scm/git` in `update` again (which feels like a better solution for now)